homeassistant/components/wiffi/binary_sensor.py

```diff
@@ -0,0 +1,60 @@
+"""Binary sensor platform support for wiffi devices."""
+
+from pathlib import Path
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from . import WiffiEntity
+from .const import CREATE_ENTITY_SIGNAL, DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up platform for a new integration.
+
+    Called by the HA framework after async_forward_entry_setup has been called
+    during initialization of a new integration (= wiffi).
+    """
+    stem = Path(__file__).stem  # stem = filename without py
+    hass.data[DOMAIN][config_entry.entry_id].async_add_entities[
```

Comment:
Nest `_create_entity` here instead so it can use the `async_add_entities` function.